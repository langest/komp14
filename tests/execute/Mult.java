class Mult {
	public static void main(String[] args) {
		int a;
		int b;
		int c;
		int d;
		int e;

		a = 10*0;
		b = 1*1;
		c = 2*1;
		d = 3*1;
		e = 2*2;

		System.out.println(a);
		System.out.println(b);
		System.out.println(b*c);
		System.out.println(b*c*d);
		System.out.println(b*c*d*e);

	}
}
