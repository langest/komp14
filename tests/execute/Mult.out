0
1
2
6
24
